nltk_contrib/timex3.py

```diff
@@ -175,9 +171,14 @@ def ground(tagged_text, base_date):
     timex_found = timex_regex.findall(tagged_text)
     timex_found = map(lambda timex:re.sub(r'</?TIMEX2.*?>', '', timex), \
                 timex_found)
+    timexList = []
```

Comment:
This new variable is used to return timex values as a list in addition to the timex tagged format.